examples/rust/tcp_inlet_and_outlet/tests/portfinder.rs

```diff
@@ -0,0 +1,15 @@
+use miette::{self, Context, IntoDiagnostic};
+use std::net::TcpListener;
+
+pub type Result<T> = miette::Result<T>;
+
+pub fn find_available_port() -> Result<u16> {
```

Comment:
You can return the same `Result` type as the other code in the examples. If you do that, you can then remove the miette dependency, as it's not really needed in this case.

Reply:
Also, you could move this function to `/Users/adrian/projects/ockam/ockam/tools/docs/example_test_helper/src/lib.rs`, which acts like a kind of "utils" file that most of the examples use.